cmd/main.go

```diff
@@ -164,18 +169,30 @@ func main() {
 	signal.Notify(c, os.Interrupt)
 
 	// subscribing to nats
-	natsClient, err := nats.New("<server-url>")
+	natsClient, err := nats.New("nats://127.0.0.1:4222")
```

Comment:
Let's externalize this reference. Ideally, we will include this reference in Meshery's system-wide settings. This is a pointer to NATS server as managed by Meshery Operator, yeah?

Reply:
Yes correct. That will also be covered in the upcoming PRs